src/uu/sort/src/check.rs

```diff
@@ -36,7 +36,7 @@ pub fn check(path: &str, settings: &GlobalSettings) -> i32 {
     for _ in 0..2 {
         recycled_sender
             .send(Chunk::new(vec![0; 100 * 1024], |_| Vec::new()))
-            .unwrap();
+            .ok();
```

Comment:
Can this fail? If not, I think unwrap is more appropriate. Turning it into an `Option` and then not doing anything with that feels a bit weird. Maybe we should return `1` if it failed?

Reply:
Yeah, this can actually fail. However, we should not care whether it fails. That `.ok()` was to make clippy happy because we otherwise don't use the result.
If it fails this means that the reader has gone away. This can happen between the first send and the second send if the reader was able to fill all the input into the first chunk we sent. So, in that case there's a race between the reader and us, if the reader exits before we attempt to send the second chunk, the send will not succeed.

I changed this to a `let _ = ` to make the intention to ignore the return value clearer.